heft-plugins/heft-localization-typings-plugin/src/schemas/options.schema.json

```diff
@@ -50,27 +50,31 @@
 
     "srcFolder": {
       "type": "string",
-      "description": "Source code root directory. Defaults to \"src/\"."
+      "description": "Source code root directory. Defaults to \"src/\".",
+      "pattern": "[^\\\\]"
```

Comment:
This pattern doesn't forbid backslashes. Rather it is "contains at least one character that is not a backslash".